deploy/stand-up-dev-env.sh

```diff
@@ -18,7 +18,8 @@ DEPLOY_PATH=${STRATOS_UI_PATH}/deploy
 
 NO_UI=false
 CLEAN=false
-
+PACKAGE_JSON_VERSION=$(cat ${STRATOS_UI_PATH}/package.json | grep version | grep -Po "([0-9\.]?)*")
```

Comment:
This does not work on Mac - grep -P not supported I think. Not sure about Windows either

Reply:
This will work on the mac for the grep command:

grep -Eo "([0-9]*\.[0-9]*\.[0-9]*)"